lib/src/webvtt/parse/parseCue.ts

```diff
@@ -0,0 +1,124 @@
+import type { WebVttCue } from '../WebVttCue.js';
+import { WebVttParsingError } from '../WebVttParsingError.js';
+import type { WebVttRegion } from '../WebVttRegion.js';
+import { parseOptions } from './parseOptions.js';
+import { parseTimeStamp } from './parseTimestamp.js';
+import { Settings } from './Settings.js';
+
+const BAD_TIMESTAMP = 'Malformed time stamp.';
+
+export function parseCue(input: string, cue: WebVttCue, regionList: WebVttRegion[]): void {
+	// Remember the original input if we need to throw an error.
+	var oInput = input;
+	// 4.1 WebVTT timestamp
+	function consumeTimeStamp() {
+		var ts = parseTimeStamp(input);
```

Comment:
(Optional) I've seen vars, should we use `const` instead?

```suggestion
	const oInput = input;
	// 4.1 WebVTT timestamp
	function consumeTimeStamp() {
		const ts = parseTimeStamp(input);
```

Similar below, I'll leave this with you

Reply:
Fixed.